verify_system.sh

```
@@ -0,0 +1,169 @@
+echo "  DOCX size: $DOCX_SIZE bytes"
+
+# Inspect the DOCX contains expected content (best-effort)
+python3 - <<PY || true
+from docx import Document
+import sys
+try:
+    doc=Document("outputs/_verif.docx")
+    full="\n".join(p.text for p in doc.paragraphs)
+    # Check if DOCX was generated and has content
+    if len(full.strip()) > 100:
+        print("OK DOCX generated with content (${#full} chars)")
```

Shellcheck (0.10.0)

[warning] full is referenced but not assigned.

(SC2154)